aspnet/web-pages/overview/ui-layouts-and-themes/twitter-helper.md

```diff
@@ -13,6 +13,9 @@ Twitter Helper with ASP.NET Web Pages
 ====================
 by [Tom FitzMacken](https://github.com/tfitzmac)
 
+> [!IMPORTANT]
+> Twitter Helpers are depreciated. For Twitter's latest engagement tools for websites, see [Twitter for Websites Overview](https://developer.twitter.com/en/docs/twitter-for-websites/overview).
```

Comment:
```suggestion
> Twitter Helpers are obsolete. For Twitter's latest engagement tools for websites, see [Twitter for Websites Overview](https://developer.twitter.com/en/docs/twitter-for-websites/overview).
```

Reply:
Per the style guide, we're supposed to avoid using "deprecated". The word "depreciated" seems awkward in this context. For that reason, I recommend "obsolete".